jsonschema/_format.py

```diff
@@ -158,7 +158,7 @@ def is_host_name(instance):
 
 
 try:
-    import rfc3987
+    import rfc3986
```

Comment:
We'd need to preserve backwards compatibility here.

So we'd need to first try to import `rfc3987.parse` and then fall back on `rfc3986.is_valid_uri`. The tests also should be updated to run under both modules (isolated from each other) to ensure that they always both pass the suite. This can be done fairly easily with a tox matrix I think. I can give some more guidance if need be.